pkgs/games/MAR1D/default.nix

```diff
@@ -0,0 +1,53 @@
+{stdenv, fetchFromGitHub, cmake, mesa_glu, x11, xorg, xinput_calibrator, doxygen, libpthreadstubs, alsaLib, alsaOss, libao}:
```

Comment:
Can you wrap this? The standard convention is either "word" wrapping at 80 chars or one per line.